src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -116,12 +116,18 @@ public CollectionItemsEqualConstraint Using<T>(IEqualityComparer<T> comparer)
             return this;
         }
 
-        #endregion
+        internal CollectionItemsEqualConstraint Using(EqualityAdapter adapter)
+        {
+            comparer.ExternalComparers.Add(adapter);
+            return this;
+        }
 
-        /// <summary>
-        /// Compares two collection members for equality
-        /// </summary>
-        protected bool ItemsEqual(object x, object y)
+#endregion
+
+    /// <summary>
+    /// Compares two collection members for equality
+    /// </summary>
+    protected bool ItemsEqual(object x, object y)
```

Comment:
Minor formatting issue, this lost an indent level and the #endregion should be indented.